clang/test/Driver/sycl-device-lib-old-model.cpp

```diff
@@ -0,0 +1,233 @@
+///
+/// Perform several driver tests for SYCL device libraries on Linux
```

Comment:
Is there a test file for tests using the new offload driver?

Reply:
For this, no.  Any device libraries or objects aren't touched by the driver with the new model, so there aren't any behaviors to verify.